.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+# Enable version updates for maven
+  - package-ecosystem: "maven"
+    directory: "/"
+    # Check for updates once a month
+    schedule:
+      interval: "monthly"
```

Comment:
Does it reasonable?

Reply:
At the initial stage, I suggest monthly, as there may be a large number of automatically generated PRs.